src/reader/sma_speedwire_reader.py

```diff
@@ -85,6 +85,17 @@ def read(self, obis_channel: int, obis_type: int, **rdg) -> bytes:
 
         return self.__get_value(self._stored_values, obis_channel, obis_type)
 
+    def scan_serials(self) -> list:
+        serials = []
+        for _ in range(self._max_responses):
+            datagram = self.__get_datagram()
+            if datagram is None:
+                break
```

Comment:
Do we want to `break` or `continue` here? Both have pros and cons. If we `break`, we might interrupt the scan prematurely, simply because no meter sent data within the timeout period (5 second default I think). If we `continue`, we might keep the scan going unnecessarily even though there are no meters sending data.

I don't know how often meters send data normally. I'm fine to go with `break` for now, and if this causes an issue where meters are not detected, we can change to `continue`.

Reply:
In my experience trying at 1-2 sites, they normally are very "chatty", 5 seconds should be enough to detect something. 
Of course I'll test it live and see if that's actually the case for couple of sites.